bootstrap.sh

```diff
@@ -55,6 +55,18 @@ program_exists() {
     fi
 }
 
+nvim_exists() {
+    local ret='0'
+    type 'nvim' >/dev/null 2>&1 || { local ret='1'; }
+
+    # throw error on non-zero return value
+    if [ ! "$ret" -eq '0' ]; then
+        return 1
+    fi
+
+    return 0
+}
+
```

Comment:
hah, yeah that was it. I must have been barely awake when I made that comment. I think I'd like to see program_exists renamed to something that indicates it's a fatal error, and then use program_exists for what you're doing with nvim_exists